tst/EnergyPlus/unit/WeatherManager.unit.cc

```diff
@@ -1145,8 +1145,144 @@ TEST_F(EnergyPlusFixture, Add_and_InterpolateWeatherInputOutputTest)
     ReadWeatherForDay(*state, 1, 1, true);
 
     // Test the feature of interpolating some weather inputs to calc sky temp
-    Real64 expected_SkyTemp = -20.8188538296;
-    EXPECT_NEAR(state->dataWeatherManager->TomorrowSkyTemp(2, 1), expected_SkyTemp, 1e-6);
+    Real64 expected_SkyTemp = -22.8763495;
```

Comment:
For the existing unit test, revise to use a variable in the second hour---instead of in the first hour---to validate, so that it would not impacted by this newly added feature (decoupled). The first hour values would be tested in the newly added unit test.